library/nats-streaming

```diff
@@ -1,34 +1,34 @@
 Maintainers: Ivan Kozlovic <ivan@synadia.com> (@kozlovic)
 GitRepo: https://github.com/nats-io/nats-streaming-docker.git
-GitCommit: 236d21372934860eb197cc4b25ad7e58b981ed6f
+GitCommit: 9716618a61deba8a7e28b69d53d95d698bde5117
 
-Tags: 0.16.2-linux, linux
-SharedTags: 0.16.2, latest
+Tags: 0.17.0-linux, linux
+SharedTags: 0.17.0, latest
 Architectures: amd64, arm32v6, arm32v7, arm64v8
-amd64-GitCommit: 236d21372934860eb197cc4b25ad7e58b981ed6f
+amd64-GitCommit: 9716618a61deba8a7e28b69d53d95d698bde5117
 amd64-Directory: amd64
-arm32v6-GitCommit: 236d21372934860eb197cc4b25ad7e58b981ed6f
+arm32v6-GitCommit: 9716618a61deba8a7e28b69d53d95d698bde5117
 arm32v6-Directory: arm32v6
-arm32v7-GitCommit: 236d21372934860eb197cc4b25ad7e58b981ed6f
+arm32v7-GitCommit: 9716618a61deba8a7e28b69d53d95d698bde5117
 arm32v7-Directory: arm32v7
-arm64v8-GitCommit: 236d21372934860eb197cc4b25ad7e58b981ed6f
+arm64v8-GitCommit: 9716618a61deba8a7e28b69d53d95d698bde5117
 arm64v8-Directory: arm64v8
 
-Tags: 0.16.2-nanoserver, nanoserver, nanoserver-1803
-SharedTags: 0.16.2, latest
+Tags: 0.17.0-nanoserver, nanoserver, nanoserver-1803
+SharedTags: 0.17.0, latest
 Architectures: windows-amd64
-windows-amd64-GitCommit: 236d21372934860eb197cc4b25ad7e58b981ed6f
+windows-amd64-GitCommit: 9716618a61deba8a7e28b69d53d95d698bde5117
 windows-amd64-Directory: windows/nanoserver-1803
 Constraints: nanoserver-1803
```

Comment:
This whole section should be removed, right? (https://github.com/nats-io/nats-streaming-docker/commit/a13ea6748fdfc3de616619ffb3b2d3b842bd1fea)

Reply:
So sorry... I was going to, got distracted and pushed before removing. I did amend the commit and "push force".